src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/ApplicationDesignerEditorFactory.vb

```diff
@@ -84,7 +84,7 @@ Namespace Microsoft.VisualStudio.Editors.ApplicationDesigner
 
                     Dim DesignerService As IVSMDDesignerService = CType(_siteProvider.GetService(GetType(IVSMDDesignerService)), IVSMDDesignerService)
                     If DesignerService Is Nothing Then
-                        Throw New Exception(SR.GetString(SR.DFX_EditorNoDesignerService, FileName))
+                        Throw New Exception(SR.GetString(My.Resources.Microsoft_VisualStudio_AppDesigner_Designer.DFX_EditorNoDesignerService, FileName))
```

Comment:
I see the examples, this is happening because SR.GetString only exists on the SR class, but the DFX_EditorNoDesignerService exists on it's base Microsoft_VisualStudio_AppDesigner_Designer. 

Basically, SR exists for compat, above it's doing absolutely nothing [other than returning the string it is passed](https://github.com/dotnet/roslyn-project-system/blob/b22cf2e196665e747a00c60d53aba0865567f329/src/Microsoft.VisualStudio.AppDesigner/Resources/Resources.vb#L81).

We should clean this mess up: https://github.com/dotnet/roslyn-project-system/issues/1056